scripts/getDependencies.xml

```diff
@@ -40,7 +40,7 @@
 				-->
 				<isset property="jtregOnZ"/>
 				<then>
-					<property name="jtregTar" value="jtreg_6_1"/>
+					<property name="jtregTar" value="jtreg_8_2"/>
```

Comment:
Do we still need jtreg_6_1? If not, please remove it completely.

Reply:
Remove jtreg_6_1 from getDependencies.pl file you meant?

Reply:
Yes, that is correct.